src/pages/hegel/reference/becoming/sublation.mdx

```diff
@@ -0,0 +1,205 @@
+means to _to keep_ or _to preserve_. On the other hand, it means _to cause to
+cease_ or _to put an end to_.
+
+But to preserve something, Hegel notes, has a negative tinge to it because to
+preserve something one must remove it from its immediacy in order to _retain_
+it; and, in so doing, the matter is essentially open to external influences. One
+might say that to keep something may mean to hold on to it or secure it, but
+thereby losing its innocence. "That which is sublated is thus something at the
+same time preserved, something that has lost its immediacy but has not come to
+nothing for that" (Hegel 2010, 82/21.94). It can be seen already how the meaning
+of keeping here cannot but entail negation. This slippage from one idea to the
+other is exactly the dialectical thread Hegel puts under the magnifying glass.
+
+Hegel praises words in language that have such double-meanings and notes the
+Latin _tollere_, which serves as the root for the English "toll" (payment) and
+"extol" (praise). In particular, he notes the deficiency with _tollere_ that its
+positive determination only goes as far as "lifting up", whereas _aufheben_
+enjoys the additional notion of preserving.
+
+However, Hegel does not directly state the _three_ meanings of sublation, even
+though they are all implied in the text.This lack of stating it expressly
+appears puzzling given that his own usage relies heavily on it.
+
+### Structure
+
+> What is sublated does not thereby turn into nothing. Nothing is the
+> _immediate_; something sublated is on the contrary something _mediated_; it is
+> something non-existent but as a result that has proceeded from a being; it
+> still _has in itself_, therefore, the _determinateness from which it derives_
+> (Hegel 2010, 81/21.94).
+
+Sublation in Hegel's philosophy primarily concerns a special state of being. It
+is a determinateness that signifies something is no longer immediately what is
+was, but is understood as a moment of a context. Despite being something
+mediated in this way, it nonetheless contains the the "_determinateness from
+which it derives_", which can be understood as a genealogy of its dialectical
+development. Put differently, in being sublated, the matter in question is no
+longer solely grasped in its own terms, but included in the terms of another.
+
+For example, to say that `being` is sublated in `becoming` means that `being` no
+longer is understood in its immediacy but as a moment of the context of
+`becoming`.
+
+But why is something sublated? Because a dialectic movement took place where
+something no longer is the something it immediately presented itself. Put
+differently, the matter at hand is not what it is. However, dialectical movement
+alone is not sufficient to understand something as sublated. `Being` vanishing
+into `nothing` does not thereby sublate `being`. To be something sublated, by
+contrast, entails a mutated state of the category that contains its immediate
+state as negated.
```

Comment:
Very good contrast to bring into relief what sublation means. But I think it moves a but fast and a new reader might miss the contrast. Maybe make things a bit more explicit.

Reply:
Ok, I've added an extra line that hopefully hammers the point home.